build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -162,6 +162,27 @@ disableTXChecksumOffload: {{ .Values.disableTXChecksumOffload }}
 # also adjust MTU to accommodate for tunnel encapsulation overhead (if applicable).
 defaultMTU: {{ .Values.defaultMTU }}
 
+# featureRateLimit defines the rate-limiting of different features. All numbers in
```

Comment:
"defines the OVS controller packet rate limits for different features"?

Reply:
Done.